parlai/scripts/distributed_train.py

```diff
@@ -84,5 +79,20 @@ def main():
         raise RuntimeError('SLURM does not appear to be installed.')
 
 
+class DistributedTrain(ParlaiScript):
+    @classmethod
+    def setup_args(cls):
+        return setup_args()
+
+    def run(self):
+        return dist_train(self.opt)
+
+
 if __name__ == '__main__':
-    main()
+    # double check we're using SLURM
+    node_list = os.environ.get('SLURM_JOB_NODELIST')
```

Comment:
could this bit be moved into the DistributedTrain.run? I want it always checked.